src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGrid.PropertyGridAccessibleObjectTests.cs

```diff
@@ -105,5 +107,141 @@ public void PropertyGridAccessibleObject_GetPropertyValue_ControlType_IsExpected
+            PropertyGrid.PropertyGridAccessibleObject accessibleObject = new(propertyGrid);
+
+            AccessibleObject expected = propertyGrid.GridViewAccessibleObject;
+
+            Assert.Equal(expected, accessibleObject.FragmentNavigate(UiaCore.NavigateDirection.FirstChild));
+        }
+
+        [WinFormsFact]
+        public void PropertyGridAccessibleObject_FragmentNavigate_LastChild_ReturnsExpected()
+        {
+            using PropertyGrid propertyGrid = new();
+            PropertyGrid.PropertyGridAccessibleObject accessibleObject = new(propertyGrid);
+
+            AccessibleObject expected = propertyGrid.HelpPaneAccessibleObject;
+
+            Assert.Equal(expected, accessibleObject.FragmentNavigate(UiaCore.NavigateDirection.LastChild));
+        }
+
+        [WinFormsFact]
+        public void PropertyGridAccessibleObject_FragmentNavigate_LastChild_ReturnsExpected_IfHelpHidden()
+        {
+            using PropertyGrid propertyGrid = new() { HelpVisible = false };
+
+            // Assign an object with commands to make CommandsPane visible
+            using Component component = new();
+            component.Site = new SiteWithMenuCommands();
+            propertyGrid.SelectedObject = component;
+
+            PropertyGrid.PropertyGridAccessibleObject accessibleObject = new(propertyGrid);
+
+            AccessibleObject expected = propertyGrid.CommandsPaneAccessibleObject;
+
+            Assert.Equal(expected, accessibleObject.FragmentNavigate(UiaCore.NavigateDirection.LastChild));
+        }
+
+        [WinFormsFact]
+        public void PropertyGridAccessibleObject_FragmentNavigate_LastChild_ReturnsExpected_IfHelpAndCommandsHidden()
+        {
+            using PropertyGrid propertyGrid = new() { HelpVisible = false, CommandsVisibleIfAvailable = false };
+            PropertyGrid.PropertyGridAccessibleObject accessibleObject = new(propertyGrid);
+
+            AccessibleObject expected = propertyGrid.GridViewAccessibleObject;
+
+            Assert.Equal(expected, accessibleObject.FragmentNavigate(UiaCore.NavigateDirection.LastChild));
+        }
+
+        internal class SiteWithMenuCommands : ISite
+        {
+            public IComponent Component { get; }
+
+            public IContainer Container { get; }
+
+            public bool DesignMode => false;
+
+            public string Name { get; set; }
+
+            public object GetService(Type serviceType)
+            {
+                return serviceType == typeof(IMenuCommandService) ? new MenuCommandService() : null;
+            }
+        }
+
+        internal class MenuCommandService : IMenuCommandService
+        {
+            public DesignerVerbCollection Verbs { get; } = new DesignerVerbCollection(new[] { new DesignerVerb("", null) });
+
+            public void AddCommand(MenuCommand command)
+            {
+                throw new NotImplementedException();
+            }
```

Comment:
I guess it's a matter of preference, but IMO it would look cleaner if all these not implemented methods were one-line:
```suggestion
            public void AddCommand(MenuCommand command) => throw new NotImplementedException();
```
What do you think?